feature/quicksettings/src/main/java/com/google/jetpackcamera/feature/quicksettings/QuickSettingsScreen.kt

```diff
@@ -152,24 +155,21 @@ private fun ExpandedQuickSettingsUi(
                         currentFacingFront = currentCameraSettings.default_front_camera
                     )
                 },
-                //TODO: Implement Set Ratio
-                /*
                 {
                     QuickSetRatio(
                         onClick = { setVisibleQuickSetting(IsExpandedQuickSetting.ASPECT_RATIO) },
                         ratio = 1,
```

Comment:
since this button expands, it has 2 purposes:
* display to user what the current aspect ratio is
* if they want to change it, tap to expand the choices to pick from

So we'll want to change the ratio value to `currentCameraSettings.aspect_ratio` so that it'll always display current active aspect ratio